### DISCUSSION TITLE
Why was the assignment of activity.DisplayName changed from HttpRequest.Path to HttpRequest.Method?

### DISCUSSION THREAD

**Discussion by wzl-bxg** [open-telemetry/opentelemetry-dotnet#5440] [Q&A]
Is it a bug that the assignment of activity.DisplayName is changed from HttpRequest.Path to HttpRequest.Method when I use the netstandard2.0 version of OpenTelemetry.Instrumentation.AspNetCore, while in the older version before1.6 it was obtained from HttpRequest.Path?

When I use this library in my .NET Core 3.1 project, all exported activities have DisplayName set as Post and Get, which is not what I want.

**before1.6：**
![Snipaste_2024-03-13_14-15-05](https://github.com/open-telemetry/opentelemetry-dotnet/assets/70325057/4feabd33-29f7-41ad-83cb-15218a246a59)

**1.7 and newer:**
![Snipaste_2024-03-13_14-15-33](https://github.com/open-telemetry/opentelemetry-dotnet/assets/70325057/1a7989dd-e778-476d-95d2-295ebfd4e1e5)

**Comment by vishweshbankwar** [comment]
@wzl-bxg - It is not a bug. Activity.DisplayName by default gets set to http method if the route information is not available. This is as per the [specification](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md)

Using the instrumentation library with `.NET Core 3.1` isn't a supported scenario. .NET Core 3.1 has reached end of life https://dotnet.microsoft.com/en-us/download/dotnet/3.1 and you should upgrade to latest supported version.
